internal/kzg/kzg.go

```diff
@@ -1,14 +0,0 @@
-package kzg
-
-import (
-	bls12381 "github.com/consensys/gnark-crypto/ecc/bls12-381"
-	"github.com/consensys/gnark-crypto/ecc/bls12-381/fr"
-)
-
-// A polynomial in lagrange form
-type Polynomial = []fr.Element
```

Comment:
Nit: Polynomial is not size constrainted, but Blob is. I guess that's fine math-wise, but if that's not useful protocol-wise, I wonder if code could be simplified if the type-system enforces its size.